share/man/man1/POST.1
.\" Automatically generated by Pod::Man 2.25 (Pod::Simple 3.20)
.\"
.\" Standard preamble:
.\" ========================================================================
.de Sp \" Vertical space (when we can't use .PP)
.if t .sp .5v
.if n .sp
..
.de Vb \" Begin verbatim text
.ft CW
.nf
.ne \\$1
..
.de Ve \" End verbatim text
.ft R
.fi
..
.\" Set up some character translations and predefined strings.  \*(-- will
.\" give an unbreakable dash, \*(PI will give pi, \*(L" will give a left
.\" double quote, and \*(R" will give a right double quote.  \*(C+ will
.\" give a nicer C++.  Capital omega is used to do unbreakable dashes and
.\" therefore won't be available.  \*(C` and \*(C' expand to `' in nroff,
.\" nothing in troff, for use with C<>.
.tr \(*W-
.ds C+ C\v'-.1v'\h'-1p'\s-2+\h'-1p'+\s0\v'.1v'\h'-1p'
.ie n \{\
.    ds -- \(*W-
.    ds PI pi
.    if (\n(.H=4u)&(1m=24u) .ds -- \(*W\h'-12u'\(*W\h'-12u'-\" diablo 10 pitch
.    if (\n(.H=4u)&(1m=20u) .ds -- \(*W\h'-12u'\(*W\h'-8u'-\"  diablo 12 pitch
.    ds L" ""
.    ds R" ""
.    ds C` ""
.    ds C' ""
'br\}
.el\{\
.    ds -- \|\(em\|
.    ds PI \(*p
.    ds L" ``
.    ds R" ''
'br\}
.\"
.\" Escape single quotes in literal strings from groff's Unicode transform.
.ie \n(.g .ds Aq \(aq
.el       .ds Aq '
.\"
.\" If the F register is turned on, we'll generate index entries on stderr for
.\" titles (.TH), headers (.SH), subsections (.SS), items (.Ip), and index
.\" entries marked with X<> in POD.  Of course, you'll have to process the
.\" output yourself in some meaningful fashion.
.ie \nF \{\
.    de IX
.    tm Index:\\$1\t\\n%\t"\\$2"
..
.    nr % 0
.    rr F
.\}
.el \{\
.    de IX
..
.\}
.\"
.\" Accent mark definitions (@(#)ms.acc 1.5 88/02/08 SMI; from UCB 4.2).
.\" Fear.  Run.  Save yourself.  No user-serviceable parts.
.    \" fudge factors for nroff and troff
.if n \{\
.    ds #H 0
.    ds #V .8m
.    ds #F .3m
.    ds #[ \f1
.    ds #] \fP
.\}
.if t \{\
.    ds #H ((1u-(\\\\n(.fu%2u))*.13m)
.    ds #V .6m
.    ds #F 0
.    ds #[ \&
.    ds #] \&
.\}
.    \" simple accents for nroff and troff
.if n \{\
.    ds ' \&
.    ds ` \&
.    ds ^ \&
.    ds , \&
.    ds ~ ~
.    ds /
.\}
.if t \{\
.    ds ' \\k:\h'-(\\n(.wu*8/10-\*(#H)'\'\h"|\\n:u"
.    ds ` \\k:\h'-(\\n(.wu*8/10-\*(#H)'\`\h'|\\n:u'
.    ds ^ \\k:\h'-(\\n(.wu*10/11-\*(#H)'^\h'|\\n:u'
.    ds , \\k:\h'-(\\n(.wu*8/10)',\h'|\\n:u'
.    ds ~ \\k:\h'-(\\n(.wu-\*(#H-.1m)'~\h'|\\n:u'
.    ds / \\k:\h'-(\\n(.wu*8/10-\*(#H)'\z\(sl\h'|\\n:u'
.\}
.    \" troff and (daisy-wheel) nroff accents
.ds : \\k:\h'-(\\n(.wu*8/10-\*(#H+.1m+\*(#F)'\v'-\*(#V'\z.\h'.2m+\*(#F'.\h'|\\n:u'\v'\*(#V'
.ds 8 \h'\*(#H'\(*b\h'-\*(#H'
.ds o \\k:\h'-(\\n(.wu+\w'\(de'u-\*(#H)/2u'\v'-.3n'\*(#[\z\(de\v'.3n'\h'|\\n:u'\*(#]
.ds d- \h'\*(#H'\(pd\h'-\w'~'u'\v'-.25m'\f2\(hy\fP\v'.25m'\h'-\*(#H'
.ds D- D\\k:\h'-\w'D'u'\v'-.11m'\z\(hy\v'.11m'\h'|\\n:u'
.ds th \*(#[\v'.3m'\s+1I\s-1\v'-.3m'\h'-(\w'I'u*2/3)'\s-1o\s+1\*(#]
.ds Th \*(#[\s+2I\s-2\h'-\w'I'u*3/5'\v'-.3m'o\v'.3m'\*(#]
.ds ae a\h'-(\w'a'u*4/10)'e
.ds Ae A\h'-(\w'A'u*4/10)'E
.    \" corrections for vroff
.if v .ds ~ \\k:\h'-(\\n(.wu*9/10-\*(#H)'\s-2\u~\d\s+2\h'|\\n:u'
.if v .ds ^ \\k:\h'-(\\n(.wu*10/11-\*(#H)'\v'-.4m'^\v'.4m'\h'|\\n:u'
.    \" for low resolution devices (crt and lpr)
.if \n(.H>23 .if \n(.V>19 \
\{\
.    ds : e
.    ds 8 ss
.    ds o a
.    ds d- d\h'-1'\(ga
.    ds D- D\h'-1'\(hy
.    ds th \o'bp'
.    ds Th \o'LP'
.    ds ae ae
.    ds Ae AE
.\}
.rm #[ #] #H #V #F C
.\" ========================================================================
.\"
.IX Title "POST 1"
.TH POST 1 "2019-02-21" "perl v5.16.3" "User Contributed Perl Documentation"
.\" For nroff, turn off justification.  Always turn off hyphenation; it makes
.\" way too many mistakes in technical documents.
.if n .ad l
.nh
.SH "NAME"
lwp\-request \- Simple command line user agent
.SH "SYNOPSIS"
.IX Header "SYNOPSIS"
.Vb 3
\& lwp\-request [\-aeEdvhx] [\-m method] [\-b <base URL>] [\-t <timeout>]
\&             [\-i <if\-modified\-since>] [\-c <content\-type>] [\-C <credentials>]
\&             [\-p <proxy\-url>] [\-o <format>] <url>...
.Ve
.SH "DESCRIPTION"
.IX Header "DESCRIPTION"
This program can be used to send requests to \s-1WWW\s0 servers and your
local file system. The request content for \s-1POST\s0 and \s-1PUT\s0
methods is read from stdin.  The content of the response is printed on
stdout.  Error messages are printed on stderr.  The program returns a
status value indicating the number of URLs that failed.
.PP
The options are:
.IP "\-m <method>" 4
.IX Item "-m <method>"
Set which method to use for the request.  If this option is not used,
then the method is derived from the name of the program.
.IP "\-f" 4
.IX Item "-f"
Force request through, even if the program believes that the method is
illegal.  The server might reject the request eventually.
.IP "\-b <uri>" 4
.IX Item "-b <uri>"
This \s-1URI\s0 will be used as the base \s-1URI\s0 for resolving all relative URIs
given as argument.
.IP "\-t <timeout>" 4
.IX Item "-t <timeout>"
Set the timeout value for the requests.  The timeout is the amount of
time that the program will wait for a response from the remote server
before it fails.  The default unit for the timeout value is seconds.
You might append \*(L"m\*(R" or \*(L"h\*(R" to the timeout value to make it minutes or
hours, respectively.  The default timeout is '3m', i.e. 3 minutes.
.IP "\-i <time>" 4
.IX Item "-i <time>"
Set the If-Modified-Since header in the request. If \fItime\fR is the
name of a file, use the modification timestamp for this file. If
\&\fItime\fR is not a file, it is parsed as a literal date. Take a look at
HTTP::Date for recognized formats.
.IP "\-c <content\-type>" 4
.IX Item "-c <content-type>"
Set the Content-Type for the request.  This option is only allowed for
requests that take a content, i.e. \s-1POST\s0 and \s-1PUT\s0.  You can
force methods to take content by using the \f(CW\*(C`\-f\*(C'\fR option together with
\&\f(CW\*(C`\-c\*(C'\fR.  The default Content-Type for \s-1POST\s0 is
\&\f(CW\*(C`application/x\-www\-form\-urlencoded\*(C'\fR.  The default Content-type for
the others is \f(CW\*(C`text/plain\*(C'\fR.
.IP "\-p <proxy\-url>" 4
.IX Item "-p <proxy-url>"
Set the proxy to be used for the requests.  The program also loads
proxy settings from the environment.  You can disable this with the
\&\f(CW\*(C`\-P\*(C'\fR option.
.IP "\-H <header>" 4
.IX Item "-H <header>"
Send this \s-1HTTP\s0 header with each request. You can specify several, e.g.:
.Sp
.Vb 4
\&    lwp\-request \e
\&        \-H \*(AqReferer: http://other.url/\*(Aq \e
\&        \-H \*(AqHost: somehost\*(Aq \e
\&        http://this.url/
.Ve
.IP "\-C <username>:<password>" 4
.IX Item "-C <username>:<password>"
Provide credentials for documents that are protected by Basic
Authentication.  If the document is protected and you did not specify
the username and password with this option, then you will be prompted
to provide these values.
.PP
The following options controls what is displayed by the program:
.IP "\-u" 4
.IX Item "-u"
Print request method and absolute \s-1URL\s0 as requests are made.
.IP "\-U" 4
.IX Item "-U"
Print request headers in addition to request method and absolute \s-1URL\s0.
.IP "\-s" 4
.IX Item "-s"
Print response status code.  This option is always on for \s-1HEAD\s0 requests.
.IP "\-S" 4
.IX Item "-S"
Print response status chain. This shows redirect and authorization
requests that are handled by the library.
.IP "\-e" 4
.IX Item "-e"
Print response headers.  This option is always on for \s-1HEAD\s0 requests.
.IP "\-d" 4
.IX Item "-d"
Do \fBnot\fR print the content of the response.
.IP "\-o <format>" 4
.IX Item "-o <format>"
Process \s-1HTML\s0 content in various ways before printing it.  If the
content type of the response is not \s-1HTML\s0, then this option has no
effect.  The legal format values are; \fItext\fR, \fIps\fR, \fIlinks\fR,
\&\fIhtml\fR and \fIdump\fR.
.Sp
If you specify the \fItext\fR format then the \s-1HTML\s0 will be formatted as
plain latin1 text.  If you specify the \fIps\fR format then it will be
formatted as Postscript.
.Sp
The \fIlinks\fR format will output all links found in the \s-1HTML\s0 document.
Relative links will be expanded to absolute ones.
.Sp
The \fIhtml\fR format will reformat the \s-1HTML\s0 code and the \fIdump\fR format
will just dump the \s-1HTML\s0 syntax tree.
.Sp
Note that the \f(CW\*(C`HTML\-Tree\*(C'\fR distribution needs to be installed for this
option to work.  In addition the \f(CW\*(C`HTML\-Format\*(C'\fR distribution needs to
be installed for \fI\-o text\fR or \fI\-o ps\fR to work.
.IP "\-v" 4
.IX Item "-v"
Print the version number of the program and quit.
.IP "\-h" 4
.IX Item "-h"
Print usage message and quit.
.IP "\-x" 4
.IX Item "-x"
Extra debugging output.
.IP "\-a" 4
.IX Item "-a"
Set text(ascii) mode for content input and output.  If this option is not
used, content input and output is done in binary mode.
.PP
Because this program is implemented using the \s-1LWP\s0 library, it will
only support the protocols that \s-1LWP\s0 supports.
.SH "SEE ALSO"
.IX Header "SEE ALSO"
lwp-mirror, \s-1LWP\s0
.SH "COPYRIGHT"
.IX Header "COPYRIGHT"
Copyright 1995\-1999 Gisle Aas.
.PP
This library is free software; you can redistribute it and/or
modify it under the same terms as Perl itself.
.SH "AUTHOR"
.IX Header "AUTHOR"
Gisle Aas <gisle@aas.no>


RELEASENOTES
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2019-03-03] XAMPP for Linux 64bit 7.3.2-1
This version of XAMPP contains the following software releases:
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2019-01-21] XAMPP for Linux 64bit 7.3.1-0
This version of XAMPP contains the following software releases:
   - Updated PHP to 7.3.1

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

[2018-12-14] XAMPP for Linux 64bit 7.3.0-0
This version of XAMPP contains the following software releases:
   - Updated PHP to 7.3.0
   - Updated MariaDB to 10.1.37
   - Updated OpenSSL 1.0.2q
   - Updated phpMyAdmin 4.8.4



htdocs/wordpress/wp-content/themes/jevelin/framework-customizations/theme/options/posts/post.php
<?php if ( ! defined( 'FW' ) ) {
	die( 'Forbidden' );
}

$options = array(

    'page_settings' => array(
        'title'   => esc_html__( 'Page Settings', 'jevelin' ),
        'type'    => 'box',
        'priority' => 'high',
        'options' => array(
            'general_tab' => array(
                'title'   => esc_html__( 'General', 'jevelin' ),
                'type'    => 'tab',
                'options' => array(

                    'post-popover' => array(
                        'label'   => esc_html__('Popover', 'jevelin'),
                        'desc'    => esc_html__('Small informative meesage', 'jevelin'),
                        'type'    => 'radio',
                        'choices' => array(
                            'none' => esc_html__('None', 'jevelin'),
                            'new' => esc_html__('NEW', 'jevelin'),
                            'hot' => esc_html__('HOT', 'jevelin'),
                        ),
                        'value' => 'none'
                    )

                ),
            ),

            'header_tab' => array(
                'title'   => esc_html__( 'Header', 'jevelin' ),
                'type'    => 'tab',
                'options' => array(


                    'header' => array(
                        'type' => 'switch',
                        'label' => esc_html__( 'Header', 'jevelin' ),
                        'desc' => esc_html__( 'Enable or disable header', 'jevelin' ),
                        'value' => 'on',
                        'left-choice' => array(
                            'value' => 'off',
                            'label' => esc_html__('Off', 'jevelin'),
                        ),
                        'right-choice' => array(
                            'value' => 'on',
                            'label' => esc_html__('On', 'jevelin'),
                        ),
                    ),

                    'header_layout' => array(
                        'type'  => 'select',
                        'value' => 'default',
                        'label' => esc_html__('Header Layout', 'jevelin'),
                        'desc'  => esc_html__('Choose main header layout', 'jevelin'),
                        'choices' => array(
                            'default' => esc_html__( 'Default (from theme options)', 'jevelin' ),
                            '1' => esc_html__( 'Header 1', 'jevelin' ),
                            '2' => esc_html__( 'Header 2', 'jevelin' ),
                            '3' => esc_html__( 'Header 3', 'jevelin' ),
                            '4' => esc_html__( 'Header 4', 'jevelin' ),
                            '5' => esc_html__( 'Header 5', 'jevelin' ),
                            '6' => esc_html__( 'Header 6 (side navigation)', 'jevelin' ),
                            '7' => esc_html__( 'Header 7 (side navigation)', 'jevelin' ),
							'8' => esc_html__( 'Header 8', 'jevelin' ),
							'9' => esc_html__( 'Header 9', 'jevelin' ),
                            'left-1' => esc_html__( 'Left Header 1', 'jevelin' ),
                            'left-2' => esc_html__( 'Left Header 2', 'jevelin' ),
                        ),
                    ),

                    'header_style' => array(
                        'type'  => 'multi-picker',
                        'label' => false,
                        'desc'  => false,
                        'value' => array(
                            'header_style' => 'default',
                            'manual' => array(
                                'description' => '',
                                'breadcrumbs' => true,
                                'scroll_button' => true,
                            ),
                        ),
                        'picker' => array(
                            'header_style' => array(
                                'type'  => 'select',
                                'value' => '1',
                                'label' => esc_html__('Style', 'jevelin'),
                                'desc'  => esc_html__('Choose main header and titlebar style', 'jevelin'),
                                'choices' => array(
                                    'default' => esc_html__( 'Default', 'jevelin' ),
                                    'light' => esc_html__( 'Light (Header + Titlebar)', 'jevelin' ),
									'light light_noborder' => esc_html__( 'Light (Header + Titlebar + withoutborder)', 'jevelin' ),
                                    'light_mobile_off' => esc_html__( 'Light (Header + Titlebar) - Mobile Off', 'jevelin' ),
                                ),
                            ),
                        ),
                        'choices' => array(
                            'light' => array(
                                'description' => array(
                                    'type'  => 'text',
                                    'value' => '',
                                    'label' => esc_html__('Description', 'jevelin'),
                                    'desc'  => esc_html__('Enter this page description', 'jevelin'),
                                ),

                                'breadcrumbs' => array(
                                    'type' => 'switch',
                                    'label' => esc_html__( 'Breadcrumbs', 'jevelin' ),
                                    'desc' => esc_html__( 'Enable or disable dreadcrumbs', 'jevelin' ),
                                    'value' => true,
                                    'left-choice' => array(
                                        'value' => false,
                                        'label' => esc_html__('Off', 'jevelin'),
                                    ),
                                    'right-choice' => array(
                                        'value' => true,
                                        'label' => esc_html__('On', 'jevelin'),
                                    ),
                                ),

                                'scroll_button' => array(
                                    'type' => 'switch',
                                    'label' => esc_html__( 'Scroll Down Button', 'jevelin' ),
                                    'desc' => esc_html__( 'Enable or disable scroll down button', 'jevelin' ),
                                    'value' => true,
                                    'left-choice' => array(
                                        'value' => false,
                                        'label' => esc_html__('Off', 'jevelin'),
                                    ),
                                    'right-choice' => array(
                                        'value' => true,
                                        'label' => esc_html__('On', 'jevelin'),
                                    ),
                                ),
                            ),
                        ),
                    ),

                    'titlebar' => array(
                        'type'  => 'select',
                        'label' => esc_html__('Titlebar', 'jevelin'),
                        'desc'  => esc_html__('Enable or disable titlebar', 'jevelin'),
                        'choices' => array(
                            'default' => esc_html__( 'Default (from theme options)', 'jevelin' ),
                            'on' => esc_html__( 'On', 'jevelin' ),
                            'off' => esc_html__( 'Off', 'jevelin' ),
                        ),
                        'value' => '',
                        'inline' => false,
                    ),

                    'titlebar_background' => array(
                        'label' => esc_html__( 'Titlebar Background Image', 'jevelin' ),
                        'desc'  => esc_html__( 'Upload titlebar background image', 'jevelin' ),
                        'type'  => 'upload'
                    ),

                    'titlebar_background_parallax' => array(
                        'type' => 'select',
                        'label' => esc_html__( 'Titlebar Parallax Background', 'jevelin' ),
                        'desc' => esc_html__( 'Enable or disable parallax background', 'jevelin' ),
                        'choices' => array(
                            'default' => esc_html__( 'Default (from theme options)', 'jevelin' ),
                            'on' => esc_html__( 'On', 'jevelin' ),
                            'off' => esc_html__( 'Off', 'jevelin' ),
                        ),
                        'value' => '',
                        'inline' => false,
                    ),


                ),
            ),

            'footer_tab' => array(
                'title'   => esc_html__( 'Footer', 'jevelin' ),
                'type'    => 'tab',
                'options' => array(

                    'footer_widgets' => array(
                        'type' => 'select',
                        'label' => esc_html__( 'Widgets', 'jevelin' ),
                        'desc' => esc_html__( 'Enable or disable footer widgets', 'jevelin' ),
                       'choices' => array(
                            'default' => esc_html__( 'Default (from theme options)', 'jevelin' ),
                            'on' => esc_html__( 'On', 'jevelin' ),
                            'off' => esc_html__( 'Off', 'jevelin' ),
                        ),
                        'value' => '',
                        'inline' => false,
                    ),

                    'copyright_bar' => array(
                        'type' => 'select',
                        'label' => esc_html__( 'Copyrights', 'jevelin' ),
                        'desc' => esc_html__( 'Enable or disable footer copyrights', 'jevelin' ),
                        'choices' => array(
                            'default' => esc_html__( 'Default (from theme options)', 'jevelin' ),
                            'on' => esc_html__( 'On', 'jevelin' ),
                            'off' => esc_html__( 'Off', 'jevelin' ),
                        ),
                        'value' => '',
                        'inline' => false,
                    ),

                ),
            ),

            'colors_tab' => array(
                'title'   => esc_html__( 'Colors', 'jevelin' ),
                'type'    => 'tab',
                'options' => array(

                    'accent_color' => array(
                        'type'  => 'color-picker',
                        'label' => esc_html__('Accent Color', 'jevelin'),
                        'desc'  => esc_html__('Set custom accent color for this page or leave blank for default', 'jevelin'),
                        'value' => '',
                    ),

                    'accent_hover_color' => array(
                        'type'  => 'rgba-color-picker',
                        'label' => esc_html__('Accent Hover Color', 'jevelin'),
                        'desc'  => esc_html__('Select page accent color on hover or leave blank for default', 'jevelin'),
                        'value' => '',
                    ),

                    'header_nav_active_color' => array(
                        'type'  => 'rgba-color-picker',
                        'label' => esc_html__('Active Navigation Color', 'jevelin'),
                        'desc'  => esc_html__('Select active navigation color or leave blank for default', 'jevelin'),
                        'value' => '',
                    ),

                ),
            ),
        ),
    ),

    'post-format-0' => array(
        'type'     => 'box',
        'title'    => esc_html__('Image Settings', 'jevelin'),
        'priority' => 'high',
        'options'  => array(

            'hide-image' => array(
                'type' => 'switch',
                'label' => esc_html__( 'Hide Featured Image', 'jevelin' ),
                'desc' => esc_html__( 'Enable this if you want to hide featured image inside the blog post', 'jevelin' ),
                'value' => false,
                'left-choice' => array(
                    'value' => false,
                    'label' => esc_html__('Off', 'jevelin'),
                ),
                'right-choice' => array(
                    'value' => true,
                    'label' => esc_html__('On', 'jevelin'),
                ),
            ),

        )
    ),

    'post-format-gallery' => array(
        'type'     => 'box',
        'title'    => esc_html__('Gallery Settings', 'jevelin'),
        'priority' => 'high',
        'options'  => array(

            'post-gallery' => array(
                'type'  => 'multi-upload',
                'label' => esc_html__( 'Gallery', 'jevelin' ),
                'desc'  => esc_html__( 'Upload images to your gallery', 'jevelin' ),
                'images_only' => true,
            ),

        )
    ),


    'post-format-quote' => array(
        'type'     => 'box',
        'title'    => esc_html__('Quote Settings', 'jevelin'),
        'priority' => 'high',
        'options'  => array(

            'post-quote' => array(
                'type'  => 'textarea',
                'label' => esc_html__( 'Quote', 'jevelin' ),
                'desc'  => esc_html__( 'Enter quote', 'jevelin' ),
            ),

            'post-quote-author' => array(
                'type' => 'text',
                'label' => esc_html__( 'Quote Author', 'jevelin' ),
                'desc' => esc_html__( 'Enter quote author', 'jevelin' ),
            ),

        )
    ),


    'post-format-link' => array(
        'type'     => 'box',
        'title'    => esc_html__('Link Settings', 'jevelin'),
        'priority' => 'high',
        'options'  => array(

            'post-url-title' => array(
                'type' => 'text',
                'label' => esc_html__( 'URL Title', 'jevelin' ),
                'desc' => esc_html__( 'Enter URL title', 'jevelin' ),
            ),

            'post-url' => array(
                'type' => 'text',
                'label' => esc_html__( 'URL', 'jevelin' ),
                'desc' => esc_html__( 'Dont forget to add http://', 'jevelin' ),
            ),

        )
    ),


    'post-format-video' => array(
        'type'     => 'box',
        'title'    => esc_html__('Video Settings', 'jevelin'),
        'priority' => 'high',
        'options'  => array(

            'post-video' => array(
                'type' => 'text',
                'label' => esc_html__( 'Video URL', 'jevelin' ),
                'desc' => esc_html__( 'Enter WordPress supported link', 'jevelin' ),
            ),

        )
    ),


    'post-format-audio' => array(
        'type'     => 'box',
        'title'    => esc_html__('Audio Settings', 'jevelin'),
        'priority' => 'high',
        'options'  => array(

            'post-audio' => array(
                'type' => 'text',
                'label' => esc_html__( 'Audio URL', 'jevelin' ),
                'desc' => esc_html__( 'Enter WordPress supported link', 'jevelin' ),
            ),

        )
    ),

);
